build.gradle

```diff
@@ -16,11 +16,29 @@ ext {
   baseVersion = "2.0"
   snapshotVersion = true
   milestone = 0
-  variants = [2.5]
+  variants = [3.0, 2.5]
   variant = System.getProperty("variant") as BigDecimal ?: variants.first()
-  if (variant == 2.5) {
+  if (variant == 3.0) {
+    groovyVersion = "3.0.0-rc-2"
+    minGroovyVersion = "3.0.0"
+    maxGroovyVersion = "3.9.99"
+    groovyDependency = [
```

Comment:
This looks like a complete duplication of the 2.5 entry, maybe move it outside, also put 2.5 before 3.0 to keep consistency

Reply:
I left it as is, as there were some changes in modules in Groovy 3 and I want to try to limit number of required dependent modules, so it may vary between versions.

Reply:
It's a trivial change to revert. I extracted that block.